sources/Application/Views/SongView.cpp

```diff
@@ -1016,3 +1011,17 @@ void SongView::OnPlayerUpdate(PlayerEventType eventType, unsigned int tick) {
 
   drawNotes();
 };
+
+void SongView::OnClockTick() {
+  // update batt gauge on every clock tick (~1Hz)
+  GUIPoint anchor = GetAnchor();
+  GUIPoint pos = anchor;
+  GUITextProperties props;
+  pos._y = 0;
+  pos._x = 26;
+
+  System *sys = System::GetInstance();
+  float batt = sys->GetBatteryLevel() / 1000.0;
+  drawBattery(batt, pos, props);
```

Comment:
Are we outputting just a number to the screen? Don't think that's ideal, people may not know what's high or low. Thought you were going for three battery levels like `[+++]`? I'd go with `[+++]`, `[++ ]`, `[+  ]` and `[   ]` with the later being "save your work!"

Reply:
Ah no, the number is not drawn, I actually did use exactly those ascii art for the gauge! :-) 
But the drawing is done in `drawBattery()` thats why I just pass in the float into it.